### PR TITLE
https://issues.redhat.com/browse/ACM-10758 Add DNS prereqs

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.9 and 2.8.

### CODE REVIEW

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -21,6 +21,18 @@ See the following prerequisites before creating a cluster in an on-premises envi
...
++
+----
+api.<cluster_name>.<base_domain>
+----
+
+- The following application base domain must point to the static IP address for Ingress VIP:
++
+----
+*.apps.<cluster_name>.<base_domain>
+----
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_cluster_on_prem.adoc:35]: Changes look good, but there are broken links
Screenshot 2024-05-10 at 10 18 56 AM

**Review by dockerymick — APPROVED** [review]
Broken links, otherwise this looks good!

File: clusters/cluster_lifecycle/create_cluster_on_prem.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_on_prem.adoc:35]: Fixed, thanks!

**Comment by oafischer** [comment]
Links fixed, merging

**Pull request merged**